Dockerfiles/cluster-agent/README.md

```diff
@@ -77,44 +95,24 @@ spec:
           - name: DD_COLLECT_KUBERNETES_EVENTS
             value: "true"
 ```
-And use the RBAC below to get the best out of it.
-
-## Pre-requisites for the DCA to interact with the API server.
-
-For the DCA to produce events, service checks and run checks one needs to enable it to perform a few actions.
-Please find the minimum RBAC listed in [the manifests](/manifests/rbac) to get the full scope of features.
-These manifests will create a Service Account, a Cluster Role with a restricted scope and actions detailed below and a Cluster Role Binding as well.
-
-### The DCA needs:
-
-- `get`, `list` and `watch` of `Componenentstatuses` to produce the controle plane service checks.
-- `get` and `update` of the `Configmaps` named `eventtokendca` to update and query the most up to date version token corresponding to the latest event stored in ETCD.
-- `watch` the `Services` to perform the Autodiscovery based off of services activity
-- `get`, `list` and `watch` of the `Pods`
-- `get`, `list` and `watch`  of the `Nodes`
-- `get`, `list` and `watch`  of the `Endpoints` to run cluster level health checks.
-
-The ConfigMap to store the `event.tokenKey` and the `event.tokenTimestamp` has to be deployed in the `default` namespace and be named `configmapdcatoken`
-One can simply run `kubectl create configmap configmapdcatoken --from-literal="event.tokenKey"="0"` .
-NB: you can set any resversion here, make sure it's not set to a value superior to the actual curent resversion.
-
-You can also set the `event.tokenTimestamp`, if not present, it will be automatically set.
+Apply the manifest:
+`kubectl create -f dca_deploy.yaml`
 
 ### Command line interface of the Cluster Agent
 
 The available commands for the Cluster Agents are:
-- `datadog-cluster-agent status`: This will give you an overview of the components of the agent and their health.
-- `datadog-cluster-agent metamap [nodeName]`: Will query the local cache of the mapping between the pods living on `nodeName`
+- `datadog-cluster-agent status`: Gives an overview of the components of the agent and their health.
```

Comment:
small nit but within the container when you display the help on `datadog-cluster-agent` the bin is named `cluster-agent`

```
Usage:
  cluster-agent [command]

Available Commands:
  flare       Collect a flare and send it to Datadog
  [...]
```